wallet/pskt/src/pskt.rs

```diff
@@ -237,6 +237,11 @@ impl PSKT<Constructor> {
         self
     }
 
+    pub fn payload(mut self, payload: Vec<u8>) -> Self {
+        self.inner_pskt.global.payload = Some(payload);
+        self
+    }
```

Comment:
Could be wise to keep the underlying signature (`Option<Vec<u8>>`), some might want to reset payload by setting a None value to it.

Reply:
I will update

Reply:
done